node/bft/src/bft.rs

```diff
@@ -518,33 +522,45 @@ impl<N: Network> BFT<N> {
                 if transmissions.contains_key(transmission_id) {
                     continue;
                 }
-                // If the transmission already exists in the ledger, skip it.
-                // Note: On failure to read from the ledger, we skip including this transmission, out of safety.
-                if self.ledger().contains_transmission(transmission_id).unwrap_or(true) {
-                    continue;
+                // Check if the ledger contains the transmission already.
+                match self.ledger().contains_transmission(transmission_id) {
+                    // On failure to read from the ledger, we skip including this transmission, out of safety.
+                    Err(err) => {
+                        warn!("{}", err.to_string());
```

Comment:
```suggestion
                        warn!("{}", err);
```
the `Display` impl (giving us `to_string`) should suffice

Reply:
2368bad08